staramr/databases/resistance/pointfinder/ARGDrugTablePointfinder.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from os import path
+
+from staramr.databases.resistance.ARGDrugTable import ARGDrugTable
+
+logger = logging.getLogger("ARGDrugTablePointfinder")
+
+"""
+A Class used to load up and search a file containing gene/drug mappings for PointFinder results.
+"""
+
+
+class ARGDrugTablePointfinder(ARGDrugTable):
+    DEFAULT_FILE = path.join(ARGDrugTable.DEFAULT_DATA_DIR, 'ARG_drug_key_pointfinder.tsv')
+
+    def __init__(self, file=DEFAULT_FILE):
+        """
+        Builds a new ARGDrugTablePointfinder from the given file.
+        :param file: The file containing the gene/drug mappings.
+        """
+        super().__init__(file=file)
+
+    def get_drug(self, organism, gene, position):
```

Comment:
Thanks for the tip.  I looked into quickly fixing this up, but it would be a larger task as I don't include all the keys I need in the data frames.  I do think it's worthwhile to do though as I think it will clean up my code quite a bit.  I've added as an issue #8

Reply:
Sounds good!